prow/config/jobs/istio.yaml

```diff
@@ -221,6 +221,12 @@ jobs:
     resources: lint
     suppress_entrypoint: true
 
+  - name: gencheck
+    command: [make, gen-check]
```

Comment:
this target doesn't exist? Even if it is optional it makes it so every PR "fails". If you really want to run this before the make target is merged can you add "hidden" to the modifiers so it doesn't show up in github?

Reply:
Good idea. Done.